src/LiveComponent/doc/index.rst

```diff
@@ -3063,6 +3063,84 @@ Then specify this new route on your component:
           use DefaultActionTrait;
       }
 
+Add a Hook on LiveProp Update
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.12
+
+    The ``onUpdated`` option was added in LiveComponents 2.12.
+
+If you want to run a custom code after a specific LiveProp was updated,
+you can do it by adding a ``onUpdated`` option. Specify a method name
+that will be called. The method should be public and created on the Component:
+
+.. code-block:: diff
+
+    // ...
+
+    #[AsLiveComponent]
+    class ProductSearch
+    {
+      - #[LiveProp(writable: true)]
+      + #[LiveProp(writable: true, onUpdated: 'onQueryUpdated')]
+        public string $query = '';
+
+        // ...
+
+        public function onQueryUpdated($previousValue): void
+        {
+            // $this->query already contains a new value
+            // and its previous value is passed as an argument
+        }
+    }
+}
+
+As soon as the `query` LiveProp is updated, the ``onQueryUpdated()`` method
+will be called. The previous value is passed there as the first argument.
+
+It also works with non-scalar types, expand ``onUpdated`` configuration
+to an array with format ``['field_name' => 'function_name']``:
+
+.. code-block::
+
+    // ...
+
+    #[AsLiveComponent]
+    class EditPost
+    {
+        #[LiveProp(writable: [LiveProp::IDENTITY, 'title', 'content'], onUpdated: ['title' => 'onTitleUpdated'])]
+        public Post $post;
+
+        // ...
+
+        public function onTitleUpdated($previousValue): void
+        {
+            // ...
+        }
+    }
+
+You can iterate more fields with corresponding function names if you want.
```

Comment:
Let's remove this entire section. This is just *such* an edge-case thing, I don't think it's even worth documenting.